packages/typespec-vscode/README.md

```diff
@@ -1,25 +1,58 @@
-# TypeSpec Language Support for VS Code
+# TypeSpec in Visual Studio Code
 
 [Repository](https://github.com/microsoft/typespec) | [Documentation](https://typespec.io/docs) | [Issues](https://github.com/microsoft/typespec/issues) | [Samples](https://github.com/microsoft/typespec/tree/main/packages/samples/specs)
 
-This provides provides [TypeSpec](https://github.com/microsoft/typespec) language support for VS Code.
+Using TypeSpec in Visual Studio Code with the extension enables efficient work. It utilizes the capabilities of VS Code to offer IntelliSense, syntax highlighting, code autocompletion, formatting, live diagnostics, rename refactoring, and go-to definition. The extension also provides a seamless experience for TypeSpec project setup and emitter configuration.
 
-![](https://raw.githubusercontent.com/microsoft/typespec/main/docs/images/vscode.gif)
+## Quick Start
 
-## Features
+- Step 1. [Install Node.js](https://nodejs.org/en/download/) and ensure you can run the [npm](https://www.npmjs.com/) command in a command promot: `npm --version`.
+- Step 2. Install the TypeSpec Compiler/CLI: `npm install -g @typespec/compiler`. Note: if the TypeSpec Compiler/CLI is not installed earlier, the extension will prompt for installation when needed.
+- Step 3. [Install the TypeSpec extension for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=typespec.typespec-vscode).
+- Step 4. Open or Create a TypeSpec project and start coding!
 
+![vscode_project_scaffolding.gif](https://raw.githubusercontent.com/microsoft/typespec/main/website/src/content/docs/docs/images/vscode_project_scaffolding.gif)
+
+## Writing TypeSpec
+
+- Auto-complete and intelliSense: Writing TypeSpec with auto-completion, code navigation, syntax highlighting
 - Live diagnostic reporting
-- Syntax highlighting
-- Code completion
-- Code folding
-- Formatting
-- Hover info
-- Rename refactoring
-- Go to definition
+- Code formating and foldering
+- Quick fixes
+- Refactorings
+
+![vscode.gif](https://raw.githubusercontent.com/microsoft/typespec/main/website/src/content/docs/docs/images/vscode.gif)
+
+## Generating from TypeSpec
+
+> **Note:**
+> - To emit dotnet SDKs, ensure to install [dotnet SDK](https://dotnet.microsoft.com/download)
+> - To emit Java SDKs, ensure to install [JDK](https://www.oracle.com/java/technologies/downloads/)
```

Comment:
We need JDK and Maven for Java:  [Maven](https://maven.apache.org/download.cgi)

Reply:
Yes! I recalled Maven is required.  But it is not mentioned in the latest [user-journey](https://github.com/allenjzhang/typespec-e2e-demo/blob/main/doc/user-journey.md#environment-setup).  BTW, it mentions only JDK, what about OpenJDK. will it work?

Reply:
Done. I've added mention of Maven